webview/package.json

```diff
@@ -11,7 +11,7 @@
     "build": "webpack --mode production",
     "test": "jest --collect-coverage",
     "storybook": "start-storybook -p 6006",
-    "build-storybook": "build-storybook",
+    "build-storybook": "build-storybook --webpack-stats-json",
     "chromatic": "npx chromatic --project-token fexq6zfy6",
```

Comment:
Do we need this line? 
We already pass it as GitHub secrets. 
https://github.com/iterative/vscode-dvc/blob/19d1c560937c2690242a2e34c8eef8fc48890763/.github/workflows/continuous-integration.yml#L56

- https://www.sabbaticaldev.co.uk/post/chromatic-storybook

Reply:
I was thinking the same thing. I don't think we need this script or even chromatic as a dependency in the sub-repo. I'll remove.